test/unit/uk/gov/hmrc/testlogin/controllers/LoginControllerSpec.scala

```diff
@@ -52,12 +52,20 @@ class LoginControllerSpec extends UnitSpec with MockitoSugar with WithFakeApplic
   }
 
   "showLoginPage" should {
+
     "display the login page" in new Setup {
 
       val result = execute(underTest.showLoginPage("/continueUrl"))
 
       bodyOf(result) should include ("Sign in")
     }
+
+    "display the Create Test User link which opens a new browser-tab" in new Setup {
```

Comment:
I think this is sufficient for our purposes - we've gone down the route of heavy selenium tests to cover this level of feature before, and the result was very much more difficult to maintain + work on.